src/www/app/app.ts

```diff
@@ -202,16 +205,14 @@ export class App {
       messageKey = 'error-unexpected';
     }
 
-    const message = messageParams ? this.localize(messageKey, ...messageParams) : this.localize(messageKey);
-
     // Defer by 500ms so that this toast is shown after any toasts that get shown when any
     // currently-in-flight domain events land (e.g. fake servers added).
     if (this.rootEl && this.rootEl.async) {
       this.rootEl.async(() => {
         this.rootEl.showToast(
-          message,
+          this.localize(messageKey, ...messageParams),
```

Comment:
Side comment, no need to fix in your PR: it would be cleaner if the if blocks above set a `message` variable, instead of messageKey and messageParams.